src/FSharpVSPowerTools.Logic/Constants.fs

```diff
@@ -55,4 +55,4 @@ let [<Literal>] cmdidPreviousHighlightedReference = 2401u
 let [<Literal>] cmdidGenerateReferencesForFsi = 0x100u
 let [<Literal>] guidAddReferenceInFSICmdSetString = "8c9a49dd-2d34-4d18-905b-c557692980be"
 let guidGenerateReferencesForFsiCmdSet = Guid(guidAddReferenceInFSICmdSetString)
-    
\ No newline at end of file
+let [<Literal>] symbolInfoMargin = "vfpt.symbolinfo.margin";
```

Comment:
I think the literal's need to start with an uppercase letter: http://stackoverflow.com/questions/3890037/literal-attribute-not-working